src/h5web/explorer/Explorer.tsx

```diff
@@ -1,76 +1,56 @@
-import { useEffect, useContext, ReactElement } from 'react';
-import { FiFileText } from 'react-icons/fi';
-import type { Entity } from '../providers/models';
+import { useContext, ReactElement, useState, Suspense } from 'react';
+import { FiFileText, FiRefreshCw } from 'react-icons/fi';
 import EntityList from './EntityList';
 import styles from './Explorer.module.css';
-import { getEntityAtPath, getParents } from '../utils';
 import { ProviderContext } from '../providers/context';
-import { isGroup } from '../guards';
-import { useSet } from 'react-use';
+import { useMount } from 'react-use';
 
 const DEFAULT_PATH = process.env.REACT_APP_DEFAULT_PATH || '/';
 
 interface Props {
-  onSelect: (entity: Entity) => void;
-  selectedEntity?: Entity;
+  onSelect: (path: string) => void;
 }
 
 function Explorer(props: Props): ReactElement {
-  const { onSelect, selectedEntity } = props;
-  const { domain, metadata: root } = useContext(ProviderContext);
+  const { onSelect } = props;
 
-  const [
-    expandedGroups,
-    { add: expandGroup, toggle: toggleGroup },
-  ] = useSet<string>();
+  const { domain } = useContext(ProviderContext);
+  const [selectedPath, setSelectedPath] = useState<string>(DEFAULT_PATH);
 
-  function handleSelect(entity: Entity): void {
-    const isExpanded = expandedGroups.has(entity.uid);
-    const isSelected = entity === selectedEntity;
-    onSelect(entity);
-
-    // Expand if collapsed; collapse is expanded and selected
-    if (isGroup(entity) && (!isExpanded || isSelected)) {
-      toggleGroup(entity.uid);
-    }
+  function handleSelect(path: string): void {
+    setSelectedPath(path);
+    onSelect(path);
   }
 
-  useEffect(() => {
-    // Find and select entity at default path
-    const entityToSelect = getEntityAtPath(root, DEFAULT_PATH) || root;
-    onSelect(entityToSelect);
-
-    // Expand entity if group
-    if (isGroup(entityToSelect)) {
-      expandGroup(entityToSelect.uid);
-    }
-
-    // Expand parent groups
-    getParents(entityToSelect).forEach((group) => expandGroup(group.uid));
-  }, [expandGroup, onSelect, root]);
+  useMount(() => {
+    onSelect(DEFAULT_PATH);
+  });
```

Comment:
I would move this under the `useState(DEFAULT_PATH)` as the two go together.

Reply:
It's gone in a subsequent PR.